src/renderer/renderScene.ts

```diff
@@ -183,10 +150,14 @@ export function renderScene(
           context.strokeRect(handler[0], handler[1], handler[2], handler[3]);
           context.lineWidth = lineWidth;
         });
-      resetZoom(context);
+      context.translate(-sceneState.scrollX, -sceneState.scrollY);
```

Comment:
We probably don't need to do the scroll translate twice, but this diff is big enough that I didn't want to touch it